src/vs/workbench/parts/tasks/electron-browser/task.contribution.ts

```diff
@@ -196,7 +196,7 @@ abstract class OpenTaskConfigurationAction extends Action {
 	}
 }
 
-class ConfigureTaskRunnerAction extends OpenTaskConfigurationAction {
+export class ConfigureTaskRunnerAction extends OpenTaskConfigurationAction {
```

Comment:
If you aren't exposing these actions from the panel, I think you can clean up these changes too.